Update docker.md to include custom admin password requirement (#8977)
opensearch-project/documentation-website: _install-and-configure/install-opensearch/docker.md

    ```
   For OpenSearch 2.12 or greater, set a new custom admin password before installation using the following command:
   ```bash

>     # Password requires a minimum 8 character and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character that is strong. Password strength can be tested here: https://lowe.github.io/tryzxcvbn

Reviewer: ```suggestion
    # Password requires a minimum of 8 characters and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character. Password strength can be tested here: https://lowe.github.io/tryzxcvbn
```
